src/internal/components/chart-plot/index.tsx

```diff
@@ -89,6 +90,7 @@ function ChartPlot(
     onBlur,
     focusOffset = DEFAULT_PLOT_FOCUS_OFFSET,
     activeElementFocusOffset = DEFAULT_ELEMENT_FOCUS_OFFSET,
+    hasHighlight,
```

Comment:
The `activeElementKey` property is used to focus the target element of the chart. Is it possible to define hasHighlight as `!!activeElementKey`?

Reply:
Did this in the [last commit](9b48f677cf9f4c40db68e5901771bbbc52d40f5f), but note that this required some further small changes because `hasHighlight` keeps its value when blurring the application whereas `activeElementKey` doesn't, so when tabbing back the entire SVG plot would be focused instead of the application. This is because in mixed charts, `activeElementKey` was only passed as true when `isPlotFocused` is true in chart container. This `isPlotFocused` tracks whether the application (not the plot, confusingly) is focused, so this part of the state is essentially a duplication of `isApplicationFocused` in chart plot. So I opted for removing `isPlotFocused` in chart container and handling this part of the state only with `isApplicationFocused` in the chart plot.

After these changes I manually tested VoiceOver in Chrome and Firefox for pie, area and bar chart.